### PR TITLE
Add a README with run steps

### PR DESCRIPTION
**Pull request by Jacalz**
This adds a README and adds a small piece of documentation to make the barrier of entry lower for people that might want help contribute to the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This repository contains the source of the main https://fyne.io website.
+
+## Running the website locally
+
+The website uses [jekyll](https://jekyllrb.com/) to generate the website from markdown files.
+With ruby installed it is as easy as running the following commands in a terminal:
+```bash
+gem install bundler jekyll
+jekyll new my-awesome-site
```
Comment (andydotxyz) [inline README.md]: I’m pretty sure this line should not be here, as the site is already initialised

**Comment by andydotxyz** [comment]
Great idea thanks. Not sure the instructions are right though.

File: README.md (hunk 1 above)
Comment (Jacalz) [reply README.md]: Yeah sorry, looks like forgot to remove that line. Will get it fixed tomorrow.

**Comment by Jacalz** [comment]
Should be fixed now. Sorry :confused:

**Comment by andydotxyz** [comment]
Great thanks. I took the liberty of squashing this one on merge, I hope that’s ok.

**Comment by Jacalz** [comment]
Perfect 👌

**Pull request merged**